src/imitation/rewards/reward_nets.py

```diff
@@ -14,148 +14,51 @@
 
 
 class RewardNet(nn.Module, abc.ABC):
-    """Abstract reward network.
-
-    Attributes:
-      observation_space: The observation space.
-      action_space: The action space.
-      use_state: should `base_reward_net` pay attention to current state?
-      use_next_state: should `base_reward_net` pay attention to next state?
-      use_action: should `base_reward_net` pay attention to action?
-      use_done: should `base_reward_net` pay attention to done flags?
-      scale: should inputs be scaled to lie in [0,1] using space bounds?
+    """Minimal abstract reward network.
+
+    Only requires the implementation of a forward pass (calculating rewards given
+    a batch of states, actions, next states and dones).
     """
 
     def __init__(
         self,
         observation_space: gym.Space,
         action_space: gym.Space,
-        *,
-        scale: bool = False,
-        use_state: bool = True,
-        use_action: bool = True,
-        use_next_state: bool = False,
-        use_done: bool = False,
+        normalize_images: bool = True,
     ):
-        """Builds a reward network.
+        """Initialize the RewardNet.
 
         Args:
-            observation_space: The observation space.
-            action_space: The action space.
-            scale: Whether to scale the input.
-            use_state: Whether state is included in inputs to network.
-            use_action: Whether action is included in inputs to network.
-            use_next_state: Whether next state is included in inputs to network.
-            use_done: Whether episode termination is included in inputs to network.
+            observation_space (gym.Space): the observation space of the environment
```

Comment:
You don't need the types in parentheses when we have type annotations. Sphinx does (or at least can -- if not we should enable it) include type annotations in the generated docs.